tests/hub/PolicyHub.Service.Tests/BusinessLogic/PolicyHubBusinessLogicTests.cs

```diff
@@ -155,7 +155,7 @@ public async Task GetPolicyContentWithFiltersAsync_WithMultipleRegexValues_Throw
         async Task Act() => await _sut.GetPolicyContentWithFiltersAsync(null, policyTypeId, "membership", OperatorId.Equals, "test");
```

Comment:
async and await are redundant, please remove (this and other occurrences)

Reply:
adjusted